### ISSUE TITLE
Empty dependencies

### ISSUE THREAD

**Issue by Brancieq-Paul** [RozeFound/mmc-export#69]
Command used:

mmc-export -i temp/mmc_prepared_export.zip -f Modrinth --modrinth-search loose -o ./output -c ./mmc-export.toml -v 1.0.4 --scheme Herobrine.fr-Optimized-{version}

Seems to work and output a .mrpack. But the dependencies are empty in modrinth.index.json:

![image](https://user-images.githubusercontent.com/61447326/214109265-3599747c-bdda-461f-bc10-88bfc9cac39f.png)

It should look like this:

![image](https://user-images.githubusercontent.com/61447326/214109413-fac5107f-1ba8-456e-b2b1-c2f81e8fba9b.png)

And I already verified, the mmc-pack.json in mmc_prepared_export.zip do contain the infos:

```json
{
    "components": [
        {
            "important": true,
            "uid": "net.minecraft",
            "version": "1.19.2"
        },
        {
            "important": true,
            "uid": "net.fabricmc.fabric-loader",
            "version": "0.14.10"
        }
    ],
    "formatVersion": 1
}
```

**Comment by Brancieq-Paul** [comment]
Found the problem.

**Comment by RozeFound** [comment]
Hmm, I see. I guess it would be better for me to use uid to identify components since it's persistent

**Comment by RozeFound** [comment]
Thanks for your report! Fixed with mmc-export 2.8.1
